slave/url_creator.py

```diff
@@ -95,19 +127,13 @@ def _artifact_to_filename(cls, platform):
     @classmethod
     def _artifact_url(cls, task_id, artifact_path, run_id=0):
         '''Return Buildbot properties for a known Buildbot generated task.'''
-        return '{}/{}/{}/{}'.format(
+        url = '{}/{}/{}/{}'.format(
             cls._artifacts,
             task_id,
             run_id,
             artifact_path)
-
-    @classmethod
-    def latest_nightly_url(cls, repo_name, product, platform):
-        '''Return URL to latest nightly build.'''
-        task_id = cls._latest_unsigned_nightly_task_id(repo_name, product, platform)
-        return cls._artifact_url(
-            task_id,
-            artifact_path='public/build/{}'.format(cls._artifact_to_filename(platform)))
+        print 'URL: %s' % url
```

Comment:
Yes, I prefer having it like this.
Debugging issues in AWFY can be a bit difficult without it.